src/main/java/school/hei/patrimoine/visualisation/swing/ihm/google/GoogleDocsLinkVerfierScreen.java

```diff
@@ -0,0 +1,206 @@
+package school.hei.patrimoine.visualisation.swing.ihm.google;
+
+import static java.awt.BorderLayout.CENTER;
+import static java.awt.Font.BOLD;
+import static java.awt.Font.PLAIN;
+import static javax.swing.SwingUtilities.invokeLater;
+import static javax.swing.WindowConstants.EXIT_ON_CLOSE;
+
+import java.awt.*;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import javax.swing.*;
+import lombok.extern.slf4j.Slf4j;
+import school.hei.patrimoine.compiler.ClassNameExtractor;
+import school.hei.patrimoine.compiler.PatrimoineCompiler;
+import school.hei.patrimoine.google.GoogleApi;
+import school.hei.patrimoine.google.GoogleApi.GoogleAuthenticationDetails;
+import school.hei.patrimoine.google.GoogleDocsLinkIdParser;
+import school.hei.patrimoine.modele.Patrimoine;
+import school.hei.patrimoine.visualisation.swing.ihm.MainIHM;
+
+@Slf4j
+public class GoogleDocsLinkVerfierScreen {
+  private final JFrame inputFrame;
+  private final JPanel inputPanel;
+  private final List<JTextField> inputFields;
+  private final GoogleDocsLinkIdInputVerifier linkIdInputVerifier =
+      new GoogleDocsLinkIdInputVerifier();
+  private final GoogleDocsLinkIdParser linkIdParser = new GoogleDocsLinkIdParser();
+  private final GoogleApi googleApi;
+  private final GoogleAuthenticationDetails authDetails;
+
+  public GoogleDocsLinkVerfierScreen(
+      GoogleApi googleApi, GoogleAuthenticationDetails authDetails, List<NamedLink> linksData) {
+    this.googleApi = googleApi;
+    this.authDetails = authDetails;
+    inputFrame = newInputFrame();
+    inputPanel = new JPanel(new GridBagLayout());
+    inputFields = new ArrayList<>();
+
+    addButtons();
+    addInputFieldsFromData(linksData);
+    configureInputFrame();
+  }
+
+  private void configureInputFrame() {
+    inputFrame.getContentPane().add(inputPanel);
+    inputFrame.setDefaultCloseOperation(EXIT_ON_CLOSE);
+    inputFrame.pack();
+    inputFrame.setLocationRelativeTo(null);
+  }
+
+  private JFrame newInputFrame() {
+    JFrame inputFrame = new JFrame("Google Docs Submission");
+    inputFrame.setSize(1200, 1000);
+    inputFrame.setResizable(true);
+    inputFrame.setVisible(true);
+    return inputFrame;
+  }
+
+  private void addButtons() {
+    JButton submitButton = newSubmitButton();
+    JButton returnButton = newReturnButton();
+
+    JLabel buttonTitle = new JLabel("Submit Your Google Docs Links:");
+    buttonTitle.setFont(new Font("Arial", BOLD, 24));
+    buttonTitle.setHorizontalAlignment(SwingConstants.CENTER);
+
+    JPanel buttonPanel = new JPanel();
+    buttonPanel.add(returnButton);
+    buttonPanel.add(submitButton);
+    buttonPanel.setOpaque(false);
+
+    GridBagConstraints gbc = new GridBagConstraints();
+    gbc.gridx = 0;
+    gbc.gridy = 0;
+    gbc.insets = new Insets(10, 0, 10, 0);
+    gbc.anchor = GridBagConstraints.CENTER;
```

Comment:
static import

Reply:
a static import cannot be used to avoid confusion with other CENTER imports